### PR TITLE
Minor misc fixes

### PR DESCRIPTION
**Pull request by kouloumos**
These fixes are some issues that I found while trying to run the setup locally. You can find more information about each fix at the respective commit.

### CODE REVIEW

File: api/.env.example [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 NODE_ENV=development
 PORT=8080
 JWT_SECRET=''
 ALBY_API_TOKEN="dkkfnfnf"
 ALBY_REDIRECT_URL="http://localhost:8080"
 ALBY_CLIENT_SECRET="secret"
-ALBY_SECRET_ID="secret"
+ALBY_CLIENT_ID="secret"
 ALBY_API_URL="https://api.alby.com"
+# We are using an LND node on voltage.cloud for staging and production
```
Comment (Extheoisah) [inline api/.env.example:10]: We shouldn't need to include comments in .env.sample

File: api/.env.example [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 LND_URL="http://localhost:8080"
 MACAROON=""
 REDIS_HOST="localhost"
 REDIS_PORT=6379
 REDIS_PASSWORD="secret"
+# The GitHub webook on the repository is how we detect that a PR
```
Comment (Extheoisah) [inline api/.env.example:16]: same here. no comments
```suggestion

```

**Review by Extheoisah — CHANGES_REQUESTED** [review]
looks good. Just a few nits

File: api/.env.example (hunk 1 above)
Comment (kouloumos) [reply api/.env.example]: You are right, I'll try to include those information in the README instead.

**Comment by kouloumos** [comment]
Removed comments from `env` file per @Extheoisah suggestion.

**Comment by Extheoisah** [comment]
> Removed comments from `env` file per @Extheoisah suggestion.

Will you be updating the README in this pr or in a later pr?

**Comment by kouloumos** [comment]
> Will you be updating the README in this pr or in a later pr?

Later

**Comment by Extheoisah** [comment]
Just realized you made the pr to main branch. Pls switch the base branch to dev

**Comment by kouloumos** [comment]
> Just realized you made the pr to main branch. Pls switch the base branch to dev

Rebased on top of `dev`.
Also added another commit with a change in seeders for transcripts

**Review by Extheoisah — APPROVED** [review]
Thanks for fixing the seeders. That's q technical debt we needed to offload

**Pull request merged**